edi_oca/models/edi_exchange_type.py

```diff
@@ -222,6 +222,11 @@ def _make_exchange_filename(self, exchange_record):
         ext = self.exchange_file_ext
         pattern = pattern + ".{ext}"
         dt = self._make_exchange_filename_datetime()
+        # Avoid duplicating type code in filename
+        if "{type.code}" in pattern and slugify(self.code) == slugify(self.name):
```

Comment:
this will be true in most cases:

![image](https://github.com/user-attachments/assets/2b7dec02-d54b-41a7-8fc1-14fc8b37aa85)

wasn't the case occurring only without a related record?

Reply:
ok, can you improve the PR description to distinct the 2 cases:
- when we have a related record, {record_name} will...
- when we don't, ..., which can lead to redundant information in the filename due to...

also need to update commit name to reflect the new implementation

Reply:
The PR is updated as suggested